pom.xml

```diff
@@ -1297,23 +1295,28 @@
             <artifactId>maven-javadoc-plugin</artifactId>
             <configuration>
               <excludePackageNames>com.acme,*.jmh</excludePackageNames>
-              <links>
-                <link>http://docs.oracle.com/javase/8/docs/api/</link>
-                <link>http://docs.oracle.com/javaee/7/api</link>
-                <link>http://junit.sourceforge.net/javadoc/</link>
-              </links>
-              <tags>
-                <tag>
-                  <name>org.apache.xbean.XBean</name>
-                  <placement>X</placement>
-                  <head />
-                </tag>
-              </tags>
             </configuration>
           </plugin>
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>aggregate-javadoc-jar</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <configuration>
+                <attach>true</attach>
+                <excludePackageNames>com.acme,*.jmh*,org.eclipse.jetty.embedded*,org.eclipse.jetty.example.asyncrest*,org.eclipse.jetty.test*</excludePackageNames>
```

Comment:
Why is this a profile?
Can't this just be a `<pluginManagement>` or `<plugin>` configuration instead?

Reply:
I went ahead and just made these configs standard